models/public/licenses/APACHE-2.0-TensorFlow.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2016 The TensorFlow Authors.  All rights reserved.
+Copyright 2019 The TensorFlow Authors.  All rights reserved.
```

Comment:
The original file is actually from the `tensorflow/models` repository. We need to retain it, but I think it would make sense to use the `APACHE-2.0-TensorFlow.txt` name for the license from the main TF repository. Could you move the original file to `APACHE-2.0-TF-Models.txt` and update any references that pointed to it?

Reply:
I created a separate PR: https://github.com/openvinotoolkit/open_model_zoo/pull/1931
New license file will be added after merging above PR and rebasing this PR.

Reply:
Excellent, thanks.